lesson4/lesson4.cpp

```diff
@@ -0,0 +1,142 @@
+#include <iostream>
+#include <algorithm>
+
+enum class Months {
+    January = 1,
+    February,
+    March,
+    April,
+    May,
+    June,
+    July,
+    August,
+    September,
+    October,
+    November,
+    December
+};
+
+int main()
+{
+    //First task
+    float a = 0.f;
+    float b = 0.f;
+    float c = 0.f;
+    std::cout << "Enter three numbers: \n";
+    std::cin >> a;
+    std::cin >> b;
+    std::cin >> c;
+
+    std::cout << "Max of these numbers is: " << std::max(a, std::max(b, c)) << std::endl;
+
+    //Second task
+    int MyNumber = 0;
+    std::cout << "Enter a number: \n";
+    std::cin >> MyNumber;
+
+    if ((MyNumber % 5 == 0) && (MyNumber % 11 == 0))
+    {
+        std::cout << "This number divides by 5 and 11 without a remainder" << std::endl;
+    }
+    else
+    {
+        std::cout << "This number cannot be divided by 5 or/and 11 without remainder" << std::endl;
+    }
+
+    //third task
+    float FirstAngle = 0.f;
+    float SecondAngle = 0.f;
+    float ThirdAngle = 0.f;
+    std::cout << "Enter values for a triangle angles: \n";
+    std::cin >> FirstAngle;
+    std::cin >> SecondAngle;
+    std::cin >> ThirdAngle;
+
+    if ( FirstAngle + SecondAngle + ThirdAngle == 180)
```

Comment:
What about negative values? zero values?

Reply:
Added handling of negative values and 0.